docs/collecting-application-metrics.md

```diff
@@ -183,42 +183,7 @@ This section coverts the following metrics modifications:
 
 ### Filtering metrics
 
-In order to filter in or out the metrics, you can add [filterprocessor] to metric's extraProcessors. Please see the following example:
```

Comment:
As a general rule, I'm wondering if it isn't better to keep these examples here and *also* link to the filtering document. That might make it easier to navigate this documentation, but I'm not entirely sure. @astencel-sumo wdyt?

Reply:
The main counterargument for that is maintainability: most of these examples I removed were already outdated. I'm not particularly for or against it, just pointing out.

Reply:
I agree with Adam that the examples should only exist in one place, not two.

I agree with Mikołaj 😄 that I'd rather see metrics filtering described in the "Collecting metrics" document and logs filtering described in the "Collecting logs" document, rather than have a separate "Filtering" document.

Reply:
That's also a valid point, though if we have a lot of examples, it'll make that document quite long.

Reply:
Do we have a decision here? I'm rather for having this in a separate file.

Reply:
OK, I see we have it already [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/08caa788ddab44f2c0e9f7cfcb485274bff2473d/docs/collecting-application-metrics.md#filtering-metrics)